pkg/importer/dataStream.go

```diff
@@ -703,6 +714,45 @@ func copy(r io.Reader, out string, qemu bool, imageSize string) error {
 	return nil
 }
 
+//ValidateSpaceConstraint - validates wheather there is enough space in PVC for both qcow2 image and converted raw image
+//Assumes it is possible to retrieve info from qcow2 file by means of qemu-img utility
+//Returns failure if either of the following happens
+//1. info cannot be retrieved from qcow2 file
+//2. Either ActualSize or VirtualSize are 0 - not specifed
+//3. ActualSize and VirtualSize together exceed available PVC Space
+func ValidateSpaceConstraint(qcow2Image string, destDir string) (bool, error) {
+
+	//Verify there is enough space in pvc before conversion
+	info, err := qemuOperations.Info(qcow2Image)
+
+	if err != nil {
+		return false, errors.Wrap(err, "Local image validation failed to retrieve image info")
+	}
+
+	if info == nil {
+		return false, errors.Wrap(err, "Local image validation failed to retrieve image info")
+	}
+
+	imgVirtualSize := info.VirtualSize
+	imgActualSize := info.ActualSize
+
+	if imgVirtualSize == 0 || imgActualSize == 0 {
```

Comment:
Any reason you can just do:
```go
if info.VirtualSize == 0 || info.ActualSize == 0 {
  ...
}

convRequiredSpace := info.VirtualSize + info.ActualSize
```
You already have the info object, this is purely a style thing, as what you have works as well.